tasks.sh

```diff
@@ -75,6 +70,12 @@ if [[ $SKIP_SYSTEMD == 1 ]]; then
   KERNEL_CMDLINE_EXTRA="init=/sbin/init-minimal $KERNEL_CMDLINE_EXTRA"
 fi
 
+# Let the user override environment variables for their special needs
```

Comment:
Thanks Juri!

From a quick look, moving this block breaks at least the SKIP_SYSTEMD logic in the block just above. I'm thinking it could make sense to just have a BUILD_DIR variable that the user could override and with a default value of ".". Then we could use it as part of the existing default MAKE command and we wouldn't have to reorder these blocks. I think it'd be a bit more inline with the existing (whacky, I agree) logic, what do you think ?

I'm on holiday for a week so I'll only be able to give this a proper look in some time, apologies in advance if I'm not responsive ;)